src/Simput/index.js

```diff
@@ -74,6 +74,7 @@ export default React.createClass({
       postJSON('/data', {
         results: results.results,
         model: results.model,
+        copies: results.copies,
```

Comment:
what about `copies: results.copies || [],` ?

Reply:
Fixed with commit 0abeb829c562b40297fbe42203890d9d3b43cee4.

Reply:
Should we do the same for  `model: results.model,` ?
Some types do not return `model` in their response.
See [demo](https://github.com/Kitware/simput/blob/master/types/demo/src/convert.js) for example.

Reply:
I would leave the model the way it is.